src/main/java/org/openapitools/api/impl/AccountsApiServiceImpl.java

```diff
@@ -24,76 +31,122 @@ public AccountsApiServiceImpl(@Nonnull Identity identity) {
 
     @Override
     public Response accountsIdGet(String username, SecurityContext securityContext) throws NotFoundException {
-        String uid = identity.getAccountService().getAccount(username).getUid();
-        String firstName = identity.getAccountService().getAccount(username).getFirstName();
-        String lastName = identity.getAccountService().getAccount(username).getLastName();
-        String email = identity.getAccountService().getAccount(username).getEmail();
-        String password = identity.getAccountService().getAccount(username).getPassword();
-        Instant createTs = identity.getAccountService().getAccount(username).getCreateTime();
-        Instant updateTs = identity.getAccountService().getAccount(username).getUpdateTime();
-        String description = identity.getAccountService().getAccount(username).getDescription();
-
-        String msg = email;
-        return Response.ok().entity(new ApiResponseMessage(ApiResponseMessage.OK, msg)).build();
+        Account account = identity.getAccountService().getAccount(username);
+        String firstName = account.getFirstName();
+        String lastName = account.getLastName();
+        String description = account.getDescription();
+
+        JSONObject jsonObject = new JSONObject();
```

Comment:
OpenAPI has already generated the model classes, you don't need to generate the json by yourself.

Reply:
Sorry, I am still confused, does that mean I can send the `Account` instance to the frontend?